opentelemetry-sdk/tests/context/propagation/test_b3_format.py

```diff
@@ -86,10 +86,12 @@ def test_extract_multi_header(self):
             new_carrier[FORMAT.SPAN_ID_KEY],
             b3_format.format_span_id(child.context.span_id),
         )
+        self.assertFalse(child.context.is_remote)
```

Comment:
I don't think this check is needed. `child` is created from a `SpanContext` extracted from the carrier, so whether `is_remote` is `False` or `True` depends on the `Span` implementation (that you are already testing in `test_span_context_remote_flag`) and *not* on the propagator one.

Same consideration for all the cases where `child.is_remote` is tested.

Reply:
That's an interesting nuance. Are we expecting that different Span SDKs will handle this behavior differently? I guess I understood it as even different Span implementations SHOULD still adhere to the span propagation behavior. 

Even the link this PR is referring to lives in the "API" section: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#spancontext

Reply:
I agree that a test checking that the `is_remote` flag is not propagated to children is useful, however this is not the place to such test, that behavior depends on the tracer implementation and not in the propagator one.